news/2018-09-17-GSoC2018.html

```diff
@@ -0,0 +1,222 @@
+                A separate panel bar for mobile displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-smallScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <div class="row">
+                <div class="columns small-12 medium-9 large-9">
+                    <header class="floe-header">
+                        <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">floe
+                            <span class="floe-header-tagline">
+                                flexible learning for open education
+                            </span></a>
+                        </h1>
+                    </header>
+                </div>
+                <div class="columns small-12 medium-3 large-3 align-self-bottom">
+                    <nav class="floe-nav">
+                        <ul>
+                            <li><a href="index.html">News</a></li>
+                            <li><a href="../resources.html">Resources</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <article id="content" class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <!-- BEGIN markup for news item -->
+                <h3>Google Summer of Code 2018</h3>
+                <time class="floe-date" datetime="2018-09-17">September 17, 2018</time>
+                <p>
+                    From March to August 2018, the Floe project participated in
```

Comment:
I don't think it is necessary to write about the GSOC process. I think it is sufficient that we participated in GSOC and that students presented their final projects were presented in a community meeting with a link to the recording.

Reply:
I think Greg was trying to get at the community engagement with this. Do you think we should remove it?

Reply:
If we want the community engagement aspect, then perhaps we should talk more about the on-ramping process and the engaging with the students before they submit proposals, and how we work with students to guide them in their proposals.

Otherwise it just sounds like an application process which I don't think gets at the community engagement.